l10n_jp_summary_invoice_sale_partner/models/res_company.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    summary_invoice_sale_partner_type = fields.Selection(
+        selection=[
+            ("partner", "Sale Order Partner"),
```

Comment:
To be a bit more explicit.
```suggestion
            ("sale_partner", "Sale Order Partner"),
```